common/predictive-text/unit_tests/in_browser/cases/worker-wordlist-integration.js

```diff
@@ -0,0 +1,66 @@
+var assert = chai.assert;
+
+/*
+ * How to run the worlist
+ */
+describe('LMLayer using the word list model', function () {
+  describe('Prediction', function () {
+    var EXPECTED_SUGGESTIONS = 3;
+    it('will predict an empty buffer', function () {
+      var lmLayer = new LMLayer();
+      var capabilities = {
+        maxLefContextCodeUnits: 32 + ~~Math.random() * 32
+      };
+
+      // We're testing many as asynchronous messages in a row.
+      // this would be cleaner using async/await syntax, but
+      // alas some of our browsers don't support it.
+      return lmLayer.initialize(
+        capabilities,
+        {
+            type: 'wordlist',
+            wordlist: __json__['wordlists/english-1000']
+        }
+      ).then(function (_actualConfiguration) {
+        return Promise.resolve();
+      }).then(function () {
+        return lmLayer.predict(zeroTransform(), atEndOfBuffer(''));
+      }).then(function (suggestions) {
+        // TODO: not super sure what to assert here!
```

Comment:
Help wanted: what should I be asserting here? Here, I'm pretending the language model is assisting in typing something starting with "the qu".

Reply:
Currently, there is no explicit difference in the code between a prediction and a correction. There are only "suggestions".

As well, I am leaving it up to the implementer of the model to give any transform that works, whether that be one that deletes to the beginning of the word, or one that simply adds the remaining characters of the word at the insertion point.